constants.js

```diff
@@ -26,7 +26,7 @@ const CHATROOM_URL = `https://api.gitter.im${CHATROOM_SUFFIX_URL}`;
 const SERVER_PREFIX_URL = 'http://127.0.0.1:5000';
 const SERVER_DEPLOY_URL = `${SERVER_PREFIX_URL}/deploy`;
 const SERVER_HOWDOI_PREFIX_URL = `${SERVER_PREFIX_URL}/howdoi?query=`;
-const JOKES_API_URL = 'http://api.icndb.com/jokes/random';
+const JOKES_API_URL =[ 'http://api.icndb.com/jokes/random','http://tambal.azurewebsites.net/joke/random'];
```

Comment:
Fix Linting, also, convert this list into a list of Objects, where we can store url and method to access the joke.